pkgs/tools/X11/hsetroot/default.nix

```
@@ -24,6 +24,10 @@ stdenv.mkDerivation rec {
     libXinerama
   ];
 
+  postPatch = lib.optionalString stdenv.isDarwin ''
```

probably more accurate, just check if it's gcc
```suggestion
  postPatch = lib.optionalString (!stdenv.isGNU) ''
```